test/constants.js

```diff
@@ -12,6 +12,7 @@ const constants = {
   SAMPLE_URL: "http://example.com/",
   SHORTEN_URL: "https://goo.gl/Us9Txw",
   TEXT_NOT_EXISTING_ON_VISIBLE_ELEMENTS: "foo",
+  TIMEOUT : 500,
```

Comment:
What about calling it `TIMEOUT_IN_MS`?

Reply:
I liked the suggestion, I will change it.

Reply:
Done.